docs/src/main/paradox/config.md

```diff
@@ -2,14 +2,19 @@
 
 ## Connection configuration
 
-Shared configuration for the connection pool is located under `akka.persistence.r2dbc.connection-factory`. You have to set at least:
+Shared configuration for the connection pool is located under `akka.persistence.r2dbc.connection-factory`.
+Selecting a database dialect is done by first assigning one of the existing dialect blocks and then overriding
+specific configuration keys to specific values for your environment:
 
```

Comment:
WDYT about pulling the pool config into its own block when we are anyway breaking config? It should still be tied to a connection-factory but it could be `connection-factory.pool`, I think it would make things easier to get (but good with config example either way)

Reply:
Didn't break out, but added better docs in 3f21e67d0b372bae7017181760cdb2e3fe76d4e5

Reply:
> WDYT about pulling the pool config into its own block

yes and no. For us it is natural to think of it as separate but for the user I'm not sure they care about what is pool and what is connection. It will never be used without the pool. So, at least I don't see a good reason for breaking the structure more than necessary.

Reply:
Alright, let's leave it as is then.